frontend/package-lock.json

```diff
@@ -0,0 +1,302 @@
+{
```

Comment:
Can we not submit this file?

Reply:
This needs to be kept under source control, I thought.

Reply:
kk, just curious why it was not committed earlier then I guess